Sources/XcodeExport/XcodeColorExporter.swift

```diff
@@ -155,15 +156,37 @@ final public class XcodeColorExporter: XcodeExporterBase {
                 XcodeAssetContents.ColorData(
                     appearances: nil,
                     color: XcodeAssetContents.ColorInfo(
-                        components: colorPair.light.toHexComponents())
+                        components: colorPair.light.toHexComponents()
+                    )
                 )
             ]
             if let darkColor = colorPair.dark {
                 colors.append(
                     XcodeAssetContents.ColorData(
-                        appearances: [XcodeAssetContents.DarkAppearance()],
+                        appearances: [XcodeAssetContents.Appearance(appearance: "luminosity", value: "dark")],
+                        color: XcodeAssetContents.ColorInfo(
+                            components: darkColor.toHexComponents()
+                        )
+                    )
+                )
+            }
+            if let lightHCColor = colorPair.lightHC {
+                colors.append(
+                    XcodeAssetContents.ColorData(
+                        appearances: [XcodeAssetContents.Appearance(appearance: "contrast", value: "high")],
+                        color: XcodeAssetContents.ColorInfo(
+                            components: lightHCColor.toHexComponents()
+                        )
+                    )
+                )
+            }
+            if let darkHCColor = colorPair.darkHC {
+                colors.append(
+                    XcodeAssetContents.ColorData(
+                        appearances: [XcodeAssetContents.Appearance(appearance: "contrast", value: "high")],
```

Comment:
You forget to add dark mode appearance here. I noticed it when try to export dark mode high contrast color.

<img width="462" alt="image" src="https://user-images.githubusercontent.com/410293/160237148-4ed81936-af20-443c-85d2-99f8aa8fa56e.png">

Reply:
Also, try to avoid code duplication in this method.